lime/explanation.py

```diff
@@ -213,3 +213,142 @@ def jsonize(x): return json.dumps(x)
         ''' % (random_id, predict_proba_js, exp_js, raw_js)
         out += u'</body></html>'
         return out
+class RegressionsExplanation(object):
```

Comment:
This class has a lot of overlap with the original Explanation. If the only difference is in as_html, it's better to add a parameter to init that specifies if the explanation is a Classification or Regression. I made ImageExplanation separate because it was very different than the others, but this seems similar enough.